[FAI-15921] Add e2e testings for Windows system in CI (#131)
faros-ai/airbyte-local-cli: airbyte-local-cli-nodejs/DEVELOPER.md

### Node Js Testing

> Please make sure you have a valid DEV Faros api key set in `FARO_API_KEY` environment variable and you have your docker on, otherwise some integration tests would fail. All tests are running against DEV so you need to take the api key in dev istead prod.

Reviewer: ```suggestion
Please make sure you have a valid DEV Faros api key set in `FAROS_API_KEY` environment variable and you have your docker running, otherwise some integration tests would fail. All tests are running against DEV so you need to create the api key in dev instead of prod.
```

Author: my english teacher :))

### CLI Executable Testing

It should be easy to run Bash testing locally, while we rely Github action to do Powershell testing.
If you want to add new e2e testing, please make sure you add to both Bash and Powershell if possible.

#### Bash Testing

For Bash shell testing, we use shellspec. Please install shellspec in your terminal to test it locally.

> Same as above, you will need to set DEV `FARO_API_KEY` environment variable and have your docker on.

Reviewer: ```suggestion
Same as above, you will need to set DEV `FAROS_API_KEY` environment variable and have your docker running.
```

Author: good catch
